### PR TITLE
locker databseSchema, controller and routes

### PR DESCRIPTION
**Pull request by matusala3**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"No tests specified\" && exit 0",
-    "start": "node server.js",
-    "dev": "nodemon server.js"
```
Comment (mussa-muna) [inline package.json]: we need this for the remote deployed server, you can still use the dev same as what start is doing.

File: package.json [hunk 2]
```diff
@@ -5,8 +5,7 @@
...
+    "start": "nodemon server.js"
```
Comment (mussa-muna) [inline package.json]:
```suggestion
```

**Review by mussa-muna — CHANGES_REQUESTED** [review]
please review

**Review by matusala3 — COMMENTED** [review]
conflict resolved

**Pull request merged**